src/Nancy/Security/SSLProxy.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Nancy.Security
+{
+    using System;
+    using System.Linq;
+
+    using Nancy.Bootstrapper;
+
+    /// <summary>
+    /// Allows a BeforeRequest hook to change Url to HTTPS if X-Forwarded-Proto header present
+    /// </summary>
+    public class SSLProxy

```

Comment:
Well I was just trying to find something similar.  I thought about
InternalConfiguration but that didn't seem the right place

On Sunday, 15 February 2015, Kristian Hellang <notifications@github.com>
wrote:

> In src/Nancy/Security/SSLProxy.cs
> <https://github.com/NancyFx/Nancy/pull/1815#discussion_r24727081>:
>
> > @@ -0,0 +1,42 @@
> > +namespace Nancy.Security
> > +{
> > +    using System;
> > +    using System.Linq;
> > +
> > +    using Nancy.Bootstrapper;
> > +
> > +    /// <summary>
> > +    /// Allows a BeforeRequest hook to change Url to HTTPS if X-Forwarded-Proto header present
> > +    /// </summary>
> > +    public class SSLProxy
>
> You mean public static class Csrf? Doesn't mean it shouldn't be static.
> Why would you create an instance of SSLProxy?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/1815/files#r24727081>.
>